litedram/phy/ddr5/commands.py

```diff
@@ -160,26 +157,38 @@ class Command(Module):
 
     # String description of 1st and 2nd edge of each command, later parsed to
     # construct the value. CS is assumed to be H for 1st edge and L for 2nd edge.
-    # TODO: Fix currently fake truth table to be consistent with the JEDEC specs for DDR5
+    # It is in countrary to the table in JEDEC specs for DDR5.
+    # We changed the order to remain consistent with rest of the code.
```

Comment:
There is no reason to invert the JEDEC specs truth table. I think that rather than changing the order here, what needs to be changed is the rest of the code, as we need to be as consistent as possible with the JEDEC specs.

When the first half of the command is executed, the CS pin should be low, while on the second half, CS should be high.

You can see the changes I have added while adjusting the simulation models to the truth table here and apply them as well in this PR: https://github.com/antmicro/litedram/commit/8eca45c79ba2616b1f6e84ad9aeba6f7155bb89d#diff-4265cf9f7db59c26f313284ab7f1dcaccf3471e4d6d949ec2ca1b1e12a7ec119L68

Reply:
Basically I re-inverted the table halves and added the correct handling of the CS pin value based on which of the half is taken into account. Also, given that in DDR5 all the commands (such as ACTIVATE) happen in once cycle only, we do not need two different commands generated for each command (e.g. ACTIVATE-1 and ACTIVATE-2 in case of LPDDR)

Reply:
done